### PR TITLE
[SMALLFIX] Document how to handle not enough space when copying files to Alluxio

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Debugging-Guide.md [hunk 1]
```diff
@@ -70,6 +70,16 @@ This error happens if this property is configured with different values across s
 (e.g., one uses the default value `NOSASL` while the other is customized to `SIMPLE`).
 Please read [Configuration-Settings](Configuration-Settings.html) for how to customize Alluxio clusters and applications.
 
+### Q: How do I copy a file larger than worker's memory size from local to Alluxio?
```
Comment (gpang) [inline docs/en/Debugging-Guide.md]: "local worker's memory", or "single worker's memory"?

**Comment by gpang** [comment]
@apc999 thanks, I left a minor comment.

File: docs/en/Debugging-Guide.md (hunk 1 above)
Comment (apc999) [inline docs/en/Debugging-Guide.md]: fixed

**Comment by apc999** [comment]
@gpang thanks for the comment. PTAL

**Comment by calvinjia** [comment]
@apc999 Should we add a pointer to this in the exception that usually comes back when the user hits this scenario?

**Comment by apc999** [comment]
@calvinjia good question. in that case, we probably want to format this question in a different way as the error is simply no more space when requesting blocks. and this is one possible scenario causing that error and we should probably talk about other common reasons (and fix if any). I am fine with either way

**Comment by apc999** [comment]
@calvinjia  PTAL. thanks

**Comment by apc999** [comment]
@gpang PTAL thanks

File: core/client/src/main/java/alluxio/client/block/RetryHandlingBlockWorkerClient.java [hunk 2]
```diff
@@ -294,25 +296,29 @@ public String call() throws AlluxioTException, TException {
...
+      throw new IOException(e);
     }
   }
 
   @Override
   public synchronized boolean requestSpace(final long blockId, final long requestBytes)
       throws IOException {
     try {
-      return retryRPC(new RpcCallableThrowsAlluxioTException<Boolean>() {
+      boolean requested = retryRPC(new RpcCallableThrowsAlluxioTException<Boolean>() {
```
Comment (gpang) [inline core/client/src/main/java/alluxio/client/block/RetryHandlingBlockWorkerClient.java]: NIT: rename `requested` to `success` or `allocated` or something like that.

File: core/common/src/main/java/alluxio/RuntimeConstants.java [hunk 3]
```diff
@@ -33,8 +33,11 @@
   public static final String ALLUXIO_JAR =
       "target/alluxio-" + VERSION + "-jar-with-dependencies.jar";
 
-  /** The URL to Alluxio documentation for this version on project web site. */
+  /** The URL of Alluxio documentation for this version on project web site. */
   public static final String ALLUXIO_DOCS_URL;
 
+  /** The URL of Alluxio debugging documentation. */
+  public static final String ALLUXIO_DEBUG_DOCS_URL = ALLUXIO_DOCS_URL + "/en/Debugging-Guide.html";
```
Comment (gpang) [inline core/common/src/main/java/alluxio/RuntimeConstants.java]: What is the ordering guarantee of the static blocks? The block above (that sets `ALLUXIO_DOCS_URL`) will always execute before this line?

**Comment by gpang** [comment]
@apc999 Thanks, I left a few comments.

File: core/client/src/main/java/alluxio/client/block/RetryHandlingBlockWorkerClient.java (hunk 2 above)
Comment (apc999) [inline core/client/src/main/java/alluxio/client/block/RetryHandlingBlockWorkerClient.java]: renamed to `sucess`

File: core/common/src/main/java/alluxio/RuntimeConstants.java (hunk 3 above)
Comment (apc999) [inline core/common/src/main/java/alluxio/RuntimeConstants.java]: according to [Java Spec](http://docs.oracle.com/javase/specs/jls/se8/html/jls-12.html#jls-12.4.2), the initializers (static block or field) are executed by text order in source code

**Comment by apc999** [comment]
@gpang PR updated. PTAL.

**Comment by gpang** [comment]
LGTM

**Pull request merged**